atomic_reactor/utils/odcs.py

```diff
@@ -120,7 +120,8 @@ def renew_compose(self, compose_id):
         :return: dict, status of compose being renewed.
         """
         logger.info("Renewing compose %d", compose_id)
-        response = self.session.patch('{}composes/{}'.format(self.url, compose_id))
+        response = self.session.patch('{}composes/{}'.format(self.url, compose_id),
+                                      {"sigkeys": sigkeys})
```

Comment:
Since this is optional in ODCS, I will change the behavior to only pass sigkeys in case they are changed to avoid ODCS to assume "unsigned"